.github/workflows/draft.yml

```
@@ -150,7 +150,7 @@ jobs:
         include:
           - os: ubuntu-24.04
             arch: amd64
-          - os: ubuntu-24.04
+          - os: ubuntu-24.04-arm
```

Is it possible to use `latest` instead of a fixed version here for more flexibility?

```suggestion
          - os: ubuntu-latest-arm
```

---

I don't believe so - the docs seem to imply it's just `ubuntu-22.04-arm` or `ubuntu-24.04-arm`

---

It looks like tomorrow (January 17th, 2025) `ubuntu-latest` will be updated to `ubuntu-24.04`, https://github.com/actions/runner-images/issues/10636 - let's wait a day and see 😄

---

Yeh of course your call. I've always avoided the `*latest` images to avoid unwanted breaking changes - though we've probably got about 2 years until `ubuntu-latest` is `26.04`